catalog/solutions/ecoassist-prolca/02.00g.P/solution.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ecoassist-prolca",
+  "name": "EcoAssist-Pro/LCA",
+  "version": "02.00g.P",

```

Comment:
I'm sorry to ask, but is this the actual format of your solution's version, or might it be some sort of typo/formatting issue? 
As of right now, the catalog expects versions to follow the SemVer syntax (e.g., `"2.0.1"`; regex: `/^[0-9]\.[0-9]\.[0-9]$/`). If your format is different, could you please provide some more information about it?
(This also applies to the file path.)
Thank you!

Reply:
Thank you for your advice.
The version format I presented is our format system.
Please wait shortly as we will consider whether it is possible to register in the format of "^[0-9]\.[0-9]\.[0-9]$".
We will consider the path as well.
Thank you.

Reply:
Dear @raimundo-henriques and @prasadt1,
As a result of internal consideration, the version notation for EcoAssist-Pro/LCA on PACT PR was decided to be "2.0.0".
Please confirm it.
Thank you.